### PR TITLE
Ignore elements outside the body

### PR DESCRIPTION
**Pull request by woahdae**
Such as title and head elements.

### CODE REVIEW

**Comment by dblock** [comment]
I like it, please add to CHANGELOG.md and fix the build (Rubocop).

**Comment by woahdae** [comment]
I'm familiar with the ruby sadism project, but not rubocop. I mean I get the idea, but it's complaining that after my change, the method is one line too long (according to community standards?), and there are 0.45 too many conditions in the method (also according to community standards?). Is the solution a refactoring? That's a bit beyond my intended involvement.

**Comment by woahdae** [comment]
Found the `.robocop_todo.yml`, that's an interesting approach. Hmmm. Well just bumped the numbers, since it's already a "todo" item.

**Comment by dblock** [comment]
And the TODO file can be auto-generated with `rubocop --auto-gen-config`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### 0.1.1 (Next)
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md]: Can you please put this back?

Comment (woahdae) [inline CHANGELOG.md]: :ok_hand:

**Comment by dblock** [comment]
Merging, thanks!

**Pull request merged**